### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][JAWS/NVDA][MAS4.2.1][IntelliJ][Wipro]:Jaws and NVDA are not annoucing the message when list is empty in Run on Web App window

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1492]
**Repro Steps:**

1. Open and Start IntelliJ IDEA.
2. Create a new project.
3. Navigate to and select created Project ->Azure->Run on web app.
4. "Run on web app" dialog will open.
5. Tab to and select Use Existing web app or create new web app radio button radio button.
6. Navigate to  "List" which is present under Before launch:Build Artifacts,Activate Tool window then observe the NVDA and JAWS Behavior

**Expected Result:**
NVDA and Jaws should announce "There are no tasks to run before launch"

**Actual Result:**
**NVDA Behavior:**
NVDA is announcing as list
**Jaws Behavior:**
List view
To move to items use the Arrow keys.

**Platform:**
IntelliJ IDEA 2017.3.2 (Community edition)
Build# IC-173.4127.27
JRE:1.8.0_152-release-1024-b8amd64
JVM: OpenJDK 64 Bit Server VM by Jet brains.
OSBuild Version: Version 1709(Os Build 16299.192)

![run on web app-listview](https://user-images.githubusercontent.com/22956130/36135940-f82e4a4a-10b3-11e8-8e6c-dd1b3ada91d7.jpg)
![if the list is empty nvda is not announcing the message present in list in run on web app](https://user-images.githubusercontent.com/22956130/36135949-08ce9abc-10b4-11e8-8709-cb594f92873b.jpg)

**Comment by Eskibear** [comment]
duplicate with #1501

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749951
